tests/tests-tax.php

```diff
@@ -265,5 +265,18 @@ public function test_download_is_exclusive_of_tax() {
 	public function test_get_payment_tax() {
 		$this->assertEquals( '0.36', edd_get_payment_tax( $this->_payment_id ) );
 	}
+	
```

Comment:
No it isn't - the unit test there is setting a value for `$current_meta['tax']`. If you do not set a value, it defaults to 0 and over-rides `_edd_payment_tax` value, and by default the tax key of `_edd_payment_meta` is not set.